ccd-definition/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -4656,5 +4657,45 @@
         "CRUD": "R"
       }
     ]
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseFieldID": "caseManagementLocation",
```

Comment:
@sankhajuria @drummondjm does this field need to be given permissions for systemupdate user?
https://github.com/hmcts/civil-ccd-definition/pull/1830/files#diff-ff4b34875c60824e1c445564d1808a8d1ef5edbffaf492e1f363a7be7763e0baL289

Reply:
yes,it should be.let me update it. Thanks for sporting it